assets/api-swagger/swagger.json

```diff
@@ -1970,6 +1970,184 @@
           }
         }
       }
+    },
+    "/translations/requests": {
+      "get": {
+        "tags": [
+          "Translation Requests"
+        ],
+        "summary": "List translation requests.",
+        "parameters": [
+          {
+            "name": "request_id",
+            "in": "query",
+            "required": false,
+            "description": "The id of the translation request.",
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "required": false,
+            "description": "Get translation requests that has one of the following statuses.",
+            "schema": {
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": [
+                      "created",
```

Comment:
Public calls can't fetch `created` requests.

Reply:
Even if they provide it in the filter?

Reply:
Well I see that it is right now, but it shouldn't. Only after the requests worker has validated and processed a request, it is marked as `pending` and that is what should decide if it is available to be fetched from public calls. `created` means the request is created from the UI but not worked on yet by the requests worker.

Reply:
Would need to change [this](https://github.com/Ometria/ometria.api.template_translations/blob/c145a68fc03ba8743f53f59e4a523afcfdea6016/ometria_api_template_translations/api/translation_requests_service.py#L189) if condition to just `if internal_call`